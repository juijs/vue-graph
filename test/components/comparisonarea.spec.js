import { mount } from '@vue/test-utils'
import GraphComparisonArea from '@/components/comparisonarea'

describe('comparisonarea.js', () => {
    it('snapshot', () => {
        const wrapper = mount(GraphComparisonArea, {
            props: {
                width: 600,
                height: 400,
                shape: 'step',
                axisStep: 5,
                axisMin: [ 0, 0 ],
                axisMax: [ 100, 20 ],
                labels: [
                    "2018/01", "", "2018/03", "", "2018/05", "", "2018/07", "", "2018/09", "", "2018/11", ""
                ],
                values: [
                    [ 15, 20, 40, 60, 45, 30, 20, 10, 10, 10, 7, 5 ],
                    [ 3, 2, 1, 4, 2, 1, 2, 4, 5, 6, 1, 0 ]
                ],
            }
        });

        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
});