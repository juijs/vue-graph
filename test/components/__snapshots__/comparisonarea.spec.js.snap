// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comparisonarea.js snapshot 1`] = `
<div
  style="position: relative; user-select: none; webkit-user-select: none;"
  unselectable="on"
>
  <svg
    buffered-rendering="dynamic"
    height="400"
    style="font-family:arial,Tahoma,verdana,classic;background:#fff"
    width="600"
  >
    <g
      transform="translate(0.5,0.5)"
    >
      <defs>
        <clippath
          id="axis-clip-id-0.0"
        >
          <rect
            height="300"
            width="500"
            x="0"
            y="0"
          />
        </clippath>
        <clippath
          id="axis-clip-rect-id-0"
        >
          <rect
            height="300"
            width="500"
            x="0"
            y="0"
          />
        </clippath>
        <clippath
          id="axis-clip-id-0.1"
        >
          <rect
            height="300"
            width="500"
            x="0"
            y="0"
          />
        </clippath>
        <clippath
          id="axis-clip-rect-id-0"
        >
          <rect
            height="300"
            width="500"
            x="0"
            y="0"
          />
        </clippath>
      </defs>
      <rect
        fill="#fff"
        fill-opacity="0"
        height="300"
        rx="0"
        ry="0"
        stroke="#fff"
        stroke-width="0"
        transform="translate(50,50)"
        width="500"
        x="0"
        y="0"
      />
      <g
        class="grid-fullblock"
        transform="translate(50,350)"
      >
        <g
          transform="translate(0,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/01
          </text>
        </g>
        <g
          transform="translate(90.9090909090909,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/03
          </text>
        </g>
        <g
          transform="translate(181.8181818181818,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/05
          </text>
        </g>
        <g
          transform="translate(272.7272727272727,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/07
          </text>
        </g>
        <g
          transform="translate(363.63636363636357,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/09
          </text>
        </g>
        <g
          transform="translate(454.54545454545445,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/11
          </text>
        </g>
        <line
          stroke="#bfbfbf"
          stroke-opacity="1"
          stroke-width="2"
          x1="0"
          x2="500"
          y1="0"
          y2="0"
        />
      </g>
      <g
        transform="translate(50,50)"
      />
      <g
        class="grid-range"
        transform="translate(50,50)"
      >
        <g
          transform="translate(0,0)"
        >
          <line
            stroke="#7977C2"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="-3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="end"
            x="-8"
            y="0"
          >
            0
          </text>
        </g>
        <g
          transform="translate(0,60)"
        >
          <line
            stroke="#7977C2"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="-3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="end"
            x="-8"
            y="0"
          >
            20
          </text>
        </g>
        <g
          transform="translate(0,120)"
        >
          <line
            stroke="#7977C2"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="-3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="end"
            x="-8"
            y="0"
          >
            40
          </text>
        </g>
        <g
          transform="translate(0,180)"
        >
          <line
            stroke="#7977C2"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="-3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="end"
            x="-8"
            y="0"
          >
            60
          </text>
        </g>
        <g
          transform="translate(0,240)"
        >
          <line
            stroke="#7977C2"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="-3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="end"
            x="-8"
            y="0"
          >
            80
          </text>
        </g>
        <g
          transform="translate(0,300)"
        >
          <line
            stroke="#7977C2"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="-3"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="end"
            x="-8"
            y="0"
          >
            100
          </text>
        </g>
        <line
          stroke="#7977C2"
          stroke-opacity="1"
          stroke-width="2"
          x1="0"
          x2="0"
          y1="0"
          y2="300"
        />
      </g>
      <g
        transform="translate(50,50)"
      />
      <g
        class="grid-panel"
        display="none"
        transform="translate(50,50)"
      >
        <rect
          fill="transparent"
          height="300"
          stroke="transparent"
          width="500"
          x="0"
          y="0"
        />
      </g>
      <rect
        fill="#fff"
        fill-opacity="0"
        height="300"
        rx="0"
        ry="0"
        stroke="#fff"
        stroke-width="0"
        transform="translate(50,50)"
        width="500"
        x="0"
        y="0"
      />
      <g
        class="grid-fullblock"
        display="none"
        transform="translate(50,350)"
      >
        <g
          transform="translate(0,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/01
          </text>
        </g>
        <g
          transform="translate(90.9090909090909,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/03
          </text>
        </g>
        <g
          transform="translate(181.8181818181818,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/05
          </text>
        </g>
        <g
          transform="translate(272.7272727272727,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/07
          </text>
        </g>
        <g
          transform="translate(363.63636363636357,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/09
          </text>
        </g>
        <g
          transform="translate(454.54545454545445,0)"
        >
          <line
            stroke="#bfbfbf"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="0"
            y1="0"
            y2="3"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="0"
            y1="0"
            y2="-300"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="middle"
            x="0"
            y="13"
          >
            2018/11
          </text>
        </g>
        <line
          stroke="#bfbfbf"
          stroke-opacity="1"
          stroke-width="2"
          x1="0"
          x2="500"
          y1="0"
          y2="0"
        />
      </g>
      <g
        class="grid-range"
        transform="translate(550,50)"
      >
        <g
          transform="translate(0,0)"
        >
          <line
            stroke="#7BBAE7"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="-500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="start"
            x="8"
            y="0"
          >
            20
          </text>
        </g>
        <g
          transform="translate(0,60)"
        >
          <line
            stroke="#7BBAE7"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="-500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="start"
            x="8"
            y="0"
          >
            16
          </text>
        </g>
        <g
          transform="translate(0,120)"
        >
          <line
            stroke="#7BBAE7"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="-500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="start"
            x="8"
            y="0"
          >
            12
          </text>
        </g>
        <g
          transform="translate(0,180)"
        >
          <line
            stroke="#7BBAE7"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="-500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="start"
            x="8"
            y="0"
          >
            8
          </text>
        </g>
        <g
          transform="translate(0,240)"
        >
          <line
            stroke="#7BBAE7"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="-500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="start"
            x="8"
            y="0"
          >
            4
          </text>
        </g>
        <g
          transform="translate(0,300)"
        >
          <line
            stroke="#7BBAE7"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1.5"
            x1="0"
            x2="3"
            y1="0"
            y2="0"
          />
          <line
            stroke="#ebebeb"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="1"
            x1="0"
            x2="-500"
            y1="0"
            y2="0"
          />
          <text
            dy="3.6666666666666665"
            fill="#333"
            font-size="11"
            font-weight="normal"
            text-anchor="start"
            x="8"
            y="0"
          >
            0
          </text>
        </g>
        <line
          stroke="#7BBAE7"
          stroke-opacity="1"
          stroke-width="2"
          x1="0"
          x2="0"
          y1="0"
          y2="300"
        />
      </g>
      <g
        transform="translate(50,50)"
      />
      <g
        class="grid-panel"
        display="none"
        transform="translate(50,50)"
      >
        <rect
          fill="transparent"
          height="300"
          stroke="transparent"
          width="500"
          x="0"
          y="0"
        />
      </g>
      <g />
      <g
        class="brush-area"
        transform="translate(50,50)"
      >
        <g>
          <path
            cursor="normal"
            d="M0,45 L22.727272727272727,45 L22.727272727272727,60 L45.45454545454545,60 L68.18181818181819,60 L68.18181818181819,120 L90.9090909090909,120 L113.63636363636364,120 L113.63636363636364,180 L136.36363636363637,180 L159.0909090909091,180 L159.0909090909091,135 L181.8181818181818,135 L204.54545454545453,135 L204.54545454545453,90 L227.27272727272725,90 L249.99999999999997,90 L249.99999999999997,60 L272.7272727272727,60 L295.4545454545454,60 L295.4545454545454,30 L318.18181818181813,30 L340.9090909090909,30 L340.9090909090909,30 L363.63636363636357,30 L386.36363636363626,30 L386.36363636363626,30 L409.090909090909,30 L431.81818181818176,30 L431.81818181818176,21 L454.54545454545445,21 L477.27272727272714,21 L477.27272727272714,15 L499.9999999999999,15"
            fill="transparent"
            stroke="#7977C2"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="2"
            style="pointer-events:stroke"
            x1="0"
            x2="499.9999999999999"
          />
        </g>
        <path
          cursor="normal"
          d="M0,45 L22.727272727272727,45 L22.727272727272727,60 L45.45454545454545,60 L68.18181818181819,60 L68.18181818181819,120 L90.9090909090909,120 L113.63636363636364,120 L113.63636363636364,180 L136.36363636363637,180 L159.0909090909091,180 L159.0909090909091,135 L181.8181818181818,135 L204.54545454545453,135 L204.54545454545453,90 L227.27272727272725,90 L249.99999999999997,90 L249.99999999999997,60 L272.7272727272727,60 L295.4545454545454,60 L295.4545454545454,30 L318.18181818181813,30 L340.9090909090909,30 L340.9090909090909,30 L363.63636363636357,30 L386.36363636363626,30 L386.36363636363626,30 L409.090909090909,30 L431.81818181818176,30 L431.81818181818176,21 L454.54545454545445,21 L477.27272727272714,21 L477.27272727272714,15 L499.9999999999999,15 L499.9999999999999,0 L0,0 Z"
          fill="#7977C2"
          fill-opacity="0.5"
          stroke="#7977C2"
          stroke-dasharray="none"
          stroke-opacity="1"
          stroke-width="0"
          style="pointer-events:stroke"
          x1="0"
          x2="499.9999999999999"
        />
      </g>
      <g />
      <g />
      <g
        class="brush-area"
        transform="translate(50,50)"
      >
        <g>
          <path
            cursor="normal"
            d="M0,255 L22.727272727272727,255 L22.727272727272727,270 L45.45454545454545,270 L68.18181818181819,270 L68.18181818181819,285 L90.9090909090909,285 L113.63636363636364,285 L113.63636363636364,240 L136.36363636363637,240 L159.0909090909091,240 L159.0909090909091,270 L181.8181818181818,270 L204.54545454545453,270 L204.54545454545453,285 L227.27272727272725,285 L249.99999999999997,285 L249.99999999999997,270 L272.7272727272727,270 L295.4545454545454,270 L295.4545454545454,240 L318.18181818181813,240 L340.9090909090909,240 L340.9090909090909,225 L363.63636363636357,225 L386.36363636363626,225 L386.36363636363626,210 L409.090909090909,210 L431.81818181818176,210 L431.81818181818176,285 L454.54545454545445,285 L477.27272727272714,285 L477.27272727272714,300 L499.9999999999999,300"
            fill="transparent"
            stroke="#7BBAE7"
            stroke-dasharray="none"
            stroke-opacity="1"
            stroke-width="2"
            style="pointer-events:stroke"
            x1="0"
            x2="499.9999999999999"
          />
        </g>
        <path
          cursor="normal"
          d="M0,255 L22.727272727272727,255 L22.727272727272727,270 L45.45454545454545,270 L68.18181818181819,270 L68.18181818181819,285 L90.9090909090909,285 L113.63636363636364,285 L113.63636363636364,240 L136.36363636363637,240 L159.0909090909091,240 L159.0909090909091,270 L181.8181818181818,270 L204.54545454545453,270 L204.54545454545453,285 L227.27272727272725,285 L249.99999999999997,285 L249.99999999999997,270 L272.7272727272727,270 L295.4545454545454,270 L295.4545454545454,240 L318.18181818181813,240 L340.9090909090909,240 L340.9090909090909,225 L363.63636363636357,225 L386.36363636363626,225 L386.36363636363626,210 L409.090909090909,210 L431.81818181818176,210 L431.81818181818176,285 L454.54545454545445,285 L477.27272727272714,285 L477.27272727272714,300 L499.9999999999999,300 L499.9999999999999,300 L0,300 Z"
          fill="#7BBAE7"
          fill-opacity="0.5"
          stroke="#7BBAE7"
          stroke-dasharray="none"
          stroke-opacity="1"
          stroke-width="0"
          style="pointer-events:stroke"
          x1="0"
          x2="499.9999999999999"
        />
      </g>
      <g />
    </g>
    <g
      transform="translate(0.5,0.5)"
    />
  </svg>
</div>
`;
